apps/docs/content/getting-started/individual-packages.mdx

```diff
@@ -2,32 +2,82 @@
 title: Individual Packages
 description: An how to guide to install Hopper UI packages individually.
 order: 3
+status: ready
 ---
 
 If you need a particluar installation, you can install Hopper UI packages individually.
 
-## Components
+<FeatureFlag flag="alpha">
+    ## Components
 
-Hopper Components are React Components, if you are using another framework, or React Native, you should use [components tokens](/tokens/getting-started/introduction) directly when developing your own components.
+    Hopper Components are React Components, if you are using another framework, or React Native, you should use [components tokens](/tokens/overview/introduction) directly when developing your own components.
+
+    ### Installation
+
+    Run one of the following commands in your terminal to install the Hopper Components package.
+
+    <PackageInstallation library="components" mod="prod" />
+
+    ### Usage
+
+    A Provider is required to use Hopper Components. Wrap your application with the HopperProvider component to use our components.
+
+    ```jsx
+    import { Button } from "@hopper-ui/components";
+
+    <HopperProvider theme="dark">
+        <Button>Hello World</Button>
+    </HopperProvider>
+    ```
+</FeatureFlag>
+
+## Tokens
 
 ### Installation
 
-Run one of the following commands in your terminal to install the Hopper Components package.
+Run the following command to install the tokens package.
 
-<PackageInstallation library="components" mod="prod" />
+<PackageInstallation library="tokens" mod="prod" />
 
 ### Usage
 
-A Provider is required to use Hopper Components. Wrap your application with the HopperProvider component to use our components.
+Import the tokens in your project by doing the following:
 
-```jsx
-import { Button } from "@hopper-ui/components";
+```css title="myfile.css"
+@import "@hopper-ui/tokens/tokens.css";
+```
 
-<HopperProvider theme="dark">
-    <Button>Hello World</Button>
-</HopperProvider>
+If your application supports a dark mode, import dark mode tokens as well:
+
+```css title="myfile.css"
+@import "@hopper-ui/tokens/dark/tokens.css";
 ```
 
+You can now use the tokens in your project by using the following syntax:
+
+```css title="myfile.css"
+.my-class {
+  background-color: var(--hop-primary-surface-weak);
+}
+```
+#### Fonts
+
+In order to use Hopper fonts in your project, import the following:
+
+```css title="myfile.css"
+@import "@hopper-ui/tokens/fonts.css";
+```
+
+Font-face declarations are now imported in your project.
+
+Failing to do so would prevent you from using [font family tokens](/tokens/semantic/typography) such as `var(--hop-font-family-primary)`.
+
+*Inter* and *ABC Favorit* variable fonts are used. This allows for a more efficient use of font files, only one font is loaded per typeface, covering all font-weights. Resulting in a smaller bundle size for all.
+
+#### Styled System
```

Comment:
I don't quite understand this section. Is it here to introduce the system style or its installation?

Reply:
As it appears under the fonts section its related to fonts. This might not be a necessary info, it was in the current usage of tokens doc : https://hopper.workleap.design/tokens/getting-started/usage

Reply:
Let's leave it and revise it later